vendor/vendor.json

```diff
@@ -1980,6 +2001,18 @@
 			"revision": "f85c78b1dd998214c5f2138155b320a4a43fbe36",
 			"revisionTime": "2017-10-30T23:38:06Z"
 		},
+		{
+			"checksumSHA1": "2LpxYGSf068307b7bhAuVjvzLLc=",
+			"path": "golang.org/x/crypto/ed25519",
```

Comment:
Shouldn't we also have an origin here?

Reply:
you're right.  I'll go through these 1 by 1